drivers/wifi/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+if(CONFIG_WIFI_WINC1500)
+  zephyr_include_directories(
+    ./
+    $ENV{ZEPHYR_BASE}/ext/hal/atmel/asf/common/components/wifi/winc1500
```

Comment:
Pretty sure this line can be omitted because we are already adding this path in 

ext/hal/atmel/asf/common/components/wifi/winc1500/CMakeLists.txt